pkg/testcase/certrotate.go

```diff
@@ -4,17 +4,16 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/rancher/distros-test-framework/factory"
 	"github.com/rancher/distros-test-framework/shared"
 
 	. "github.com/onsi/gomega"
 )
 
-func TestCertRotate(cluster *factory.Cluster) {
+func TestCertRotate(cluster *shared.Cluster) {
 	certRotate(cluster.Config.Product, cluster.ServerIPs)
 
 	ip, manageError := shared.ManageService(cluster.Config.Product, "restart", "agent", cluster.AgentIPs)
-	Expect(manageError).NotTo(HaveOccurred(), fmt.Sprintf("error restarting agent node ip %s", ip))
+	Expect(manageError).NotTo(HaveOccurred(), "error restarting agent node ip"+ip)
```

Comment:
Should add a space here

Reply:
no need here, the compiler automatically does that 
![Screenshot 2024-08-05 at 16 08 44](https://github.com/user-attachments/assets/377e65e8-62ad-459a-a84d-7f05c0f02420)